app/src/main/java/com/cody/androidexercise/ui/iofile/IoFileFragment.kt

```diff
@@ -48,14 +48,13 @@ class IoFileFragment : Fragment() {
             IoFileWriteResult.Ongoing -> binding.barLoading.visibility = View.VISIBLE
 
             IoFileWriteResult.Success -> {
+                viewModel.content.value = ""
```

Comment:
set the content value in viewmodel, do not update the viewmodel variable values in fragment or activity